cli/kata-env.go

```diff
@@ -23,7 +23,7 @@ import (
 //
 // XXX: Increment for every change to the output format
 // (meaning any change to the EnvInfo type).
-const formatVersion = "1.0.12"
+const formatVersion = "1.0.15"
```

Comment:
oh, maybe I bumped that too far! I think I took the wrong line from the merge conflict. Let me wind that back to 1.0.13 and re-push in a minute..